SU2_CFD/src/solver_direct_mean_inc.cpp

```diff
@@ -3512,7 +3566,51 @@ void CIncEulerSolver::ImplicitEuler_Iteration(CGeometry *geometry, CSolver **sol
   
 }
 
-void CIncEulerSolver::SetPrimitive_Gradient_GG(CGeometry *geometry, CConfig *config) {
+void CIncEulerSolver::ComputeUnderRelaxationFactor(CSolver **solver_container, CConfig *config) {
+  
+  /* Loop over the solution update given by relaxing the linear
+   system for this nonlinear iteration. */
+  
+  su2double localUnderRelaxation = 1.0;
+  const su2double allowableRatio = 0.2;
```

Comment:
Btw these 20%, is it your experience or is it from a source?

Reply:
Both. There are a few papers using similar approaches that I can point you to, if interested. Folks seem to use something in the 20-30% range. For the turbulence variables (actually, just SA), I did not limit it as much, since the turbulence variables tend to change rapidly during the initial transients of a calculation

Reply:
You already provided the source from Mavriplis above. If you have some other feel free to post it